includes/Block/class-issue.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
This is very duplicate of class-profile.php. Construct a class-block.php base class and inherit from that.

Reply:
This has been done now.